elements/bulbs-header/bulbs-header.js

```diff
@@ -0,0 +1,65 @@
+import { registerElement, BulbsHTMLElement } from 'bulbs-elements/register';
+import './bulbs-header-masthead';
+import './bulbs-header-responsive-nav';
+import './bulbs-header.scss';
+
+import { InViewMonitor } from 'bulbs-elements/util';
+
+class BulbsHeader extends BulbsHTMLElement {
+  createdCallback () {
+    this.mastheads = this.getElementsByTagName('bulbs-header-masthead');
+    this.responsiveNavs = this.getElementsByTagName('bulbs-header-responsive-nav');
```

Comment:
I'd say we kill the use of `getElementsByTagName` and the getters if it's unclear what they're doing.